nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +127,41 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(graph, slider, min_in, max_in, value) {
+    /** check if click was the middle mouse button */
+    var target = slider;
+    var self = graph;
+    var slider_index = target.slider.index; // Get index (For 1D > sliders)
+    var x_pos = target.getAttribute('fixed-x'); //important for 2d sliders
+
+    //Math for mapping one range to another
+    var left_min = min_in;
+    var left_max = max_in;
+    var left_span = left_max - left_min;
+    var right_min = self.scale.range()[1];
+    var right_max = self.scale.range()[0];
+    var right_span = right_max - right_min;
+    var scale = (value - left_min) / left_span;
+    var point = right_min + (scale * right_span);
```

Comment:
Use the slider.scale object to do this mapping, rather than writing it all yourself.  :)